### DISCUSSION TITLE
How to ssh to the VM created by rancher-desktop?

### DISCUSSION THREAD

**Discussion by yzhao1012** [rancher-sandbox/rancher-desktop#2324] [Q&A] [answered]
Often we need to examine the VM's environment for system behaviors.
But it's not clear how to SSH to the VM created by rancher-desktop from rancher-desktop's UI.

**Comment by jandubois** [comment, marked as answer]
There is no way to do it from the GUI, but from the shell you can run `rdctl shell` to get an interactive shell, or `rdctl shell $CMD` to run a single command inside the VM.

On Linux and macOS this is equivalent to running `limactl shell rancher-desktop ...` after setting up the correct `PATH` and `LIMA_HOME`. On Windows it uses `wsl -d rancher-desktop ...`.

**Comment by jandubois** [comment]
See also https://github.com/pixie-io/pixie/issues/447#issuecomment-1145078839
